docs/src/modules/developing/pages/development-process-java.adoc

```diff
@@ -30,7 +30,7 @@ TIP: Kickstart a project using the xref:java:kickstart.adoc[Java code generation
 [#_implement_components]
 == Implement components
 
-Stateful services can implement https://developer.lightbend.com/docs/akka-serverless/reference/glossary.html#value_entity[Value Entity] or https://developer.lightbend.com/docs/akka-serverless/reference/glossary.html#event_sourced_entity[Event Sourced Entity] components. Stateless services implement Actions. Typically, a stateful service should only have one Entity type, but can also include Actions.
+Stateful services can implement https://developer.lightbend.com/docs/akka-serverless/reference/glossary.html#value_entity[Value Entity] or https://developer.lightbend.com/docs/akka-serverless/reference/glossary.html#event_sourced_entity[Event Sourced Entity] and View components. Stateless services implement Actions. Typically, a stateful service should only have one Entity type, but can also include Actions and Views.
 
```

Comment:
@pvlugter  some mention of Replicated Entity should be added here

Reply:
Yes, and Replicated Entity needs an entry in the main docs glossary.